code/system/application/legacy/vanishingpoint/apps/ImageProcessingTestFixture.cpp

```diff
@@ -0,0 +1,85 @@
+#include <iostream>
```

Comment:
Please add a short License header and a few lines of explanation for the purpose of this file.